metricbeat/include/list.go

```diff
@@ -1,33 +1,18 @@
 /*
-
-This file is included in the main file to load all metricsets.
-
-In case only a subset of metricsets should be included, they can be specified manually in the main.go file.
-
+Package include imports all Module and MetricSet packages so that they register
+their factories with the global registry. This package can be imported in the
+main package to automatically register all of the standard supported Metricbeat
+modules.
 */
 package include
 
-// Make sure all active plugins are loaded
-// TODO: create a script to automatically generate this list
```

Comment:
Lets leave it out until it bugs us, or we have the first errors hitting us here.